namespaces/live-1.cloud-platform.service.justice.gov.uk/offender-events-dev/00-namespace.yaml

```diff
@@ -9,4 +9,7 @@ metadata:
     cloud-platform.justice.gov.uk/business-unit: "HMPPS"
     cloud-platform.justice.gov.uk/application: "offender-events"
     cloud-platform.justice.gov.uk/owner: "Digital Prison Services: dps-hmpps@digital.justice.gov.uk"
-    cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/offender-events.git"
+    cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/offender-events.git, https://github.com/ministryofjustice/probation-offender-events.git, https://github.com/ministryofjustice/offender-events-ui.git"
```

Comment:
almost, https://github.com/ministryofjustice/offender-events should be https://github.com/ministryofjustice/prison-offender-events